src/components/editor/baseURL.jsx

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018 Stanford University see LICENSE for license
+
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+// Renders the base URL after Save & Publish
+const BaseURL = (props) => {
```

Comment:
i'm not in love with this name -- to me, this is like the first part of an API url, before the API specific args are added.  Also, this component is showing a div with a message containing the URI (not URL) from trellis.

How about:

ResourceUriMessage

.. and changing the action and reducer method names as well?

Reply:
Yeah, I went back and forth on that, the return value from save is "saveBaseURL", so I went with this for consistency, but started out with "ResourceURI". Happy to switch back for clarity.

Reply:
if you want to keep "baseURL" maybe "BaseUriMessage" ?  But I think all of the usage of baseURL was arbitrarily chosen and ripe for change to "resourceUri".  Just my opinion, tho.

Reply:
Renamed to ResourceURIMessage :-)